src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/WorkloadSet.cs

```diff
@@ -71,7 +71,12 @@ public static WorkloadSet FromDictionaryForJson(IDictionary<string, string> dict
         public static WorkloadSet FromJson(string json, SdkFeatureBand defaultFeatureBand)
         {
 #if USE_SYSTEM_TEXT_JSON
-            return FromDictionaryForJson(JsonSerializer.Deserialize<IDictionary<string, string>>(json)!, defaultFeatureBand);
+            var jsonSerializerOptions = new JsonSerializerOptions()
```

Comment:
I'm wondering if we should have a "standard" JsonSerializerOptions that we use everywhere to avoid forgetting AllowTrailingCommas. What do you think?